foundation-extensions/mobile-core/rules-engine/README.md

```diff
@@ -4,7 +4,7 @@ The Rules Engine is provided by the Adobe Experience Platform Mobile SDKs as par
 
 In the Experience Platform Mobile SDKs, the Rules Engine enables you to integrate the data and functionality of marketing and ad tech to help disparate products communicate. The Rules Engine looks for the user interaction and associated data, and when the criteria that you defined in the rules are met, the actions that you specified are triggered.
 
-The Rules Engine, with [Experience Platform Launch](https://experience.adobe.com/#/data-collection/)'s tag management system, allows for complex if/then/else workflows that can combine multiple solution and extension related behaviors. You can define and capture the necessary user data and orchestrate how each solution contributes to the user's experience.
+The Rules Engine, with [Data Collection UI](https://experience.adobe.com/#/data-collection/)'s tag management system, allows for complex if/then/else workflows that can combine multiple solution and extension related behaviors. You can define and capture the necessary user data and orchestrate how each solution contributes to the user's experience.
```

Comment:
Sorry!

I talked to the writer in charge, and this should be phrased as this instead: "“The Rules Engine, when implemented through tags in the Data Collection UI, allows for…”

Apologies for the confusion, but the whole current naming schema is quite... confusing, as you can see 😅.

Reply:
Got it. Done.